src/pal/pal_posix.h

```diff
@@ -151,10 +151,10 @@ namespace snmalloc
       constexpr int SIZE = 1024;
       void* buffer[SIZE];
       auto nptrs = backtrace(buffer, SIZE);
-      fflush(stdout);
+      UNUSED(fsync(STDOUT_FILENO));
```

Comment:
I would sooner this one be inside error after the three calls to write.  Then you have the pattern some output followed by a flush.  Then it is easier to see you have all the flushes.